tests/librarything-search-index-conflicts.unit.php

```
@@ -0,0 +1,154 @@
+        if ($line === '' || $line[0] === '#' || !str_contains($line, '=')) {
+            continue;
+        }
+        [$k, $v] = explode('=', $line, 2);
+        $v = trim($v);
+        if (strlen($v) >= 2 && ($v[0] === '"' || $v[0] === "'") && $v[-1] === $v[0]) {
+            $v = substr($v, 1, -1);
+        }
+        $env[trim($k)] = $v;
+    }
+    return $env;
+}
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 14. (undefined)

(ErrorControlOperator)